app/views/trainees/placements/delete.html.erb

```diff
@@ -15,7 +15,8 @@
     <h1 class="govuk-heading-l">Are you sure you want to delete this placement?</h1>
 
     <div class="govuk-body">
-      <p><%= @placement_form.placement.name %></p>
+      <p class="govuk-body govuk-!-margin-bottom-2"><%= @placement_form.placement.name %></p>
```

Comment:
The extra CSS class here makes the spacing the same as on the summary card (which styles a `dd` element that we don't have here).